libass/ass_fontselect.c

```diff
@@ -166,6 +170,111 @@ static ASS_FontProviderFuncs ft_funcs = {
     .destroy_font      = destroy_font_ft,
 };
 
+#if defined(_WIN32) && !defined(__CYGWIN__)
+/*
+ * Output parameters are only valid if the call was sucessful, ignore them otherwise.
+ * \param path          input; path of the dir to open, may be UTF-8 or the codepage used by fopen
+ * \param file_context  output
+ * \param find_handle   output; needs to be closed
+ * \param out_length    output
+ * \return utf-16le version of path; needs to be freed
```

Comment:
`\return wide-version of path or NULL on failure; needs to be freed if non-NULL`